scripts/common.sh

```diff
@@ -324,3 +324,24 @@ function gnu_grep
         grep "$@"
     fi
 }
+
+function time_json_to_file
```

Comment:
Maybe rename to something like: `time_to_json_file` or `record_cmd_time_to_json`?

Reply:
Done.